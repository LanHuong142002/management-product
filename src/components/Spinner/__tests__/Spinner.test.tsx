import { render } from '@testing-library/react';

import { Spinner } from '@components';

describe('Testing spinner', () => {
  it('Should render component', () => {
    const { container } = render(<Spinner />);

    expect(container).toMatchSnapshot();
  });
});
