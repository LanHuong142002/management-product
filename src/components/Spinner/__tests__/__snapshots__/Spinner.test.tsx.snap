// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing spinner Should render component 1`] = `
<div>
  <div
    class="spinner-wrapper"
  >
    <div
      class="spinner"
    />
  </div>
</div>
`;
